src/components/nav-menu/nav-menu.jsx

```diff
@@ -0,0 +1,46 @@
+import * as React from "react";
+import { Link } from "gatsby";
+
+import PropTypes from "prop-types";
+
+import * as navMenuStyled from "./nav-menu.module.scss";
+
+const NavMenu = () => {
+  const menuElements = [
+    {label: "Work", path:"/"},
+    {label: "Services", path:"/" },
+    {label: "Our Company", path:"/" },
+    {label: "Community", path:"/" },
+  ];
+  return (
+    menuElements.map(({label, path}) => {
+      let commonProps = {
+        sx: { variant: "navlink" }, 
```

Comment:
Can you remove this prop or is necessary?

Reply:
Done